mlibrary/pride
# Pride.FieldTree.tokens

# Overview
`Pride.FieldTree.tokens` was an array that was only ever used `Pride.FieldTree.tokenize`. This pull request moves the code into `Pride.FieldTree.tokenize`, and removes the need for `Pride.FieldTree.tokens`. `./src/Pride/FieldTree/tokens.js` and `./spec/Pride/FieldTree/tokens.spec.js` have been deleted.

## Testing
1. Build the repository (`npm run build`).
2. Run the tests to make sure they pass (`npm run test`).
   * Break the new/updated unit tests to make sure they're working properly.
3. Navigate to your local [Search repository](https://github.com/mlibrary/search) and apply the newly generated files:
   1. Open `./package.json` and add `#Pride-FieldTree-tokens` at the end of the `pride` dependency URL:
      ```bash
      "pride": "git+https://github.com/mlibrary/pride.git#Pride-FieldTree-tokens"
      ``` 
   2. Do a clean install of Search:
      ```bash
      rm -rf node_modules && rm package-lock.json && npm install
      ``` 
   3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
- Make sure the PR is consistent in these browsers:
